scripts/deploy/09_eth_put_theta_vault.ts

```diff
@@ -0,0 +1,163 @@
+import { run } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import {
+  CHAINID,
+  WETH_ADDRESS,
+  ETH_USDC_POOL,
+  USDC_ADDRESS,
+  USDC_PRICE_ORACLE,
+  ETH_PRICE_ORACLE,
+  OptionsPremiumPricerInStables_BYTECODE,
+} from "../../constants/constants";
+import OptionsPremiumPricerInStables_ABI from "../../constants/abis/OptionsPremiumPricerInStables.json";
+import {
+  AUCTION_DURATION,
+  AVAX_STRIKE_STEP,
+  ETH_STRIKE_STEP,
+  MANAGEMENT_FEE,
+  PERFORMANCE_FEE,
+  PREMIUM_DISCOUNT,
+  STRIKE_DELTA,
+} from "../utils/constants";
+
+const TOKEN_NAME = {
+  [CHAINID.ETH_MAINNET]: 'Ribbon USDC Theta Vault ETH Put',
+  [CHAINID.ETH_KOVAN]: 'Ribbon USDC Theta Vault ETH Put',
+  [CHAINID.AVAX_MAINNET]: 'Ribbon USDC Theta Vault AVAX Put',
+  [CHAINID.AVAX_FUJI]: 'Ribbon USDC Theta Vault AVAX Put',
+};
+
+const TOKEN_SYMBOL = {
+  [CHAINID.ETH_MAINNET]: 'rUSDC-ETH-P-THETA',
+  [CHAINID.ETH_KOVAN]: 'rUSDC-ETH-P-THETA',
+  [CHAINID.AVAX_MAINNET]: 'rUSDC-AVAX-P-THETA',
+  [CHAINID.AVAX_FUJI]: 'rUSDC-AVAX-P-THETA',
+};
+
+const STRIKE_STEP = {
+  [CHAINID.ETH_MAINNET]: ETH_STRIKE_STEP,
+  [CHAINID.ETH_KOVAN]: ETH_STRIKE_STEP,
+  [CHAINID.AVAX_MAINNET]: AVAX_STRIKE_STEP,
+  [CHAINID.AVAX_FUJI]: AVAX_STRIKE_STEP,
+};
+
+const main = async ({
+  network,
+  deployments,
+  ethers,
+  getNamedAccounts,
+}: HardhatRuntimeEnvironment) => {
+  const { BigNumber } = ethers;
+  const { deploy } = deployments;
+  const { deployer, owner, keeper, admin, feeRecipient } =
+    await getNamedAccounts();
+  console.log(`09 - Deploying ETH Put Theta Vault on ${network.name}`);
+
+  const chainId = network.config.chainId;
+
+  const manualVolOracle = await deployments.get("ManualVolOracle");
+  const underlyingOracle = ETH_PRICE_ORACLE[chainId];
+  const stablesOracle = USDC_PRICE_ORACLE[chainId];
+
+  const pricer = await deploy("OptionsPremiumPricerETH", {
```

Comment:
[Same with this](https://github.com/ribbon-finance/ribbon-v2/pull/198/files#r774311442)

```solidity
const pricer = await deployments.get('OptionsPremiumPricerETH')
```

Reply:
Done